webpack.config.babel.js

```diff
@@ -181,7 +181,7 @@ const basePlugins = (bundle_name) => [
       // ref: https://en.wikipedia.org/wiki/Base32
       // NOTE: please leave the BASE_32_VERSION be! It is updated automatically by
       // the release script 🤖
-      BASE_32_VERSION: 'BN',
```

Comment:
Is there any way to define this version somewhere else and reuse it here and in `release/githubActions/workflows.config`?

Reply:
We've had a discussion a while back about actually trying to move away from this since we risk overriding an already released version on S3 now that we have LTS alongside regular releases.